### ISSUE TITLE
Limit the space of cloud save of certain users

### ISSUE THREAD

**Issue by nintiendo1** [LANCommander/LANCommander#85]
It would be interesting to control whether or not a user can use cloud saving and how much space they have available for that cloud saving.
Thanks.

**Comment by pathartl** [comment]
Good idea. Won't make it by 1.0.0, but keeping this open for future development tracking.

**Comment by pathartl** [comment]
🤖 **Board Manager — Preliminary Triage**

**Component classification:** `Server`

This issue has been added to the [Development Tracker](https://github.com/orgs/LANCommander/projects/1).

*This comment was added automatically. Status will be updated as commits land.*
